admin_guide/scheduling/index.adoc

```diff
@@ -0,0 +1,38 @@
+[[admin-guide-scheduling-index]]
+= Overview
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+
+== Overview
+
+Pod scheduling is an internal process that determines how new
+pods are placed onto nodes within the cluster
+
+[[admin-guide-scheduling-index-default]]
+== Default scheduling
+
+{product-title} comes with a default scheduler that should serve the needs of most users. The default scheduler uses both inherent and customizable tools to determine the best fit for a pod. 
+
+For information on how the default scheduler determines pod placement and on customizable parameters, see xref:../../admin_guide/scheduling/scheduler.adoc#admin-guide-scheduler[Default Scheduling].
+
+[[admin-guide-scheduling-index-advanced]]
+== Advanced scheduling
+
+In situations where you might want more control over where new pods are placed, the {product-title} advanced scheduling features allow you to configure a pod so that the pod is required to (or has a preference to) run on a particular node or alongside a specific pod. Advanced scheduling also allows you to prevent pods from being placed on a node or with another pod.  
+
+For information about advanced scheduling, see xref:../../admin_guide/scheduling/scheduler-advanced.adoc#admin-guide-scheduling-advanced[Advanced Scheduling].
+
+[[admin-guide-scheduling-index-custom]]
+== Custom schedulers
+
+In addition, {product-title} allows you to use your own or third-party schedulers simply by editing the pod specification. 
```

Comment:
{product-title} _also_ allows you to use your own or third-party schedulers by editing the pod specification. (added also, removed In addition, and simply)